sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ChangeFeedMode.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.cosmos.implementation.changefeed.common;
+package com.azure.cosmos.models;
 
 import com.azure.cosmos.util.Beta;
```

Comment:
INCREMENTAL -> LATEST_VERSION_ONLY / LATEST_VERSION_IF_EXISTS (does not contain delete operations) 
FULL_FIDELITY -> ALL_VERSIONS / ALL_MODIFICATIONS / ALL_VERSIONS_AND_DELETES (because this includes deletes)

Confusion with All modifications -> if it is going to return all modified properties or just some or just the whole document. 

Above is initial feedback from customer on the naming.